bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python3
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+
+import os
+import sys
+import subprocess
+import argparse
+import json
+
+parser = argparse.ArgumentParser(
+    description="You must setup: An AWS auth means, eg. AWS_PROFILE env var. "
+    "Postgres access details, eg. DATABASE_URL env var."
+)
+parser.add_argument(
+    "--mode",
+    default="uat",
+    required=False,
+    help="Mode to run in, eg 'production, 'uat', etc (default: '%(default)s')",
+)
+parser.add_argument(
+    "--cached",
+    default=False,
+    required=False,
+    action="store_true",
+    help="Use keys generated on a previous non-cached run (default: %(default)s) "
+    "(Don't mix PRODUCTION and STAGING!)",
+)
+parser.add_argument(
+    "--pgclient",
+    default="/usr/bin/psql",
+    required=False,
+    help=f"Postgres client path (default: %(default)s)",
+)
+parser.add_argument(
+    "--awsclient",
+    default="/usr/local/bin/aws",
+    required=False,
+    help=f"AWS client path (default: %(default)s)",
+)
+args = parser.parse_args()
+
+# Globals
+AWSCLIENT = args.awsclient
+PGCLIENT = args.pgclient
+DATABASE_URL = os.getenv("DATABASE_URL", "")
+NEW_ENV = os.environ.copy()
```

Comment:
question: do we need this? I think we could just use `osv.environ` and co directly, which'd make things a bit smaller

(fwiw I don't think you ever needed it, since modifying the environment is always a local process thing, and there was nothing we needed to keep hidden from subprocesses)

Reply:
I'll give it a try, and .... `os.environ` on its own works.
Done.